docs/src/app/(public)/(content)/react/handbook/customization/page.mdx

```diff
@@ -0,0 +1,124 @@
+# Customization
+
+<Subtitle>A guide to customizing the behavior of Base UI components.</Subtitle>
+<Meta name="description" content="A guide to customizing the behavior of Base UI components." />
+
+## Base UI events
+
+Change events such as `onOpenChange`, `onValueChange`, and `onPressedChange` are custom to Base UI rather than the DOM.
+They can be emitted by various different DOM events, effects, or even during rendering.
+
+```js title="Base UI event signatures"
+onOpenChange: (open, eventDetails) => void
+onValueChange: (value, eventDetails) => void
+onPressedChange: (pressed, eventDetails) => void
+```
+
+The `eventDetails` property is passed as a second argument to Base UI event handlers.
+This enables the event to be customized, and also allows you to conditionally run side effects based on the reason or DOM event that caused the change.
+
+```tsx title="eventDetails object"
+interface BaseUIEventDetails {
+  reason: string;
+  event: Event;
+  cancel: () => void;
+  allowPropagation: () => void;
+  isCanceled: boolean;
+  isPropagationAllowed: boolean;
+}
+```
+
+- `reason` is used to determine why the change event occurred, which can be useful to conditionally run certain side effects.
+  Most IDEs will show the possible string values after typing `reason === '`.
+- `event` is the native DOM event that caused the change.
+- `cancel` stops the component from changing its internal state.
+- `allowPropagation` allows the DOM event to propagate in cases where Base UI will stop the propagation.
+- `isCanceled` indicates whether the change event has been canceled.
+- `isPropagationAllowed` indicates whether the DOM event is allowed to propagate.
+
+### Canceling a Base UI event
+
+To leave a component uncontrolled, the event can be canceled with the `cancel()` method:
```

Comment:
If the goal of this section is to say that you can influence the internal state of a component without having to control this state, then I don't think the sentence is super clear.


```suggestion
You can cancel updates without having to control the updates state thanks to the `cancel()` method:
```

I'm not convinced by my proposal either, but I think there is room for improvement here

Reply:
I tweaked the wording a bit